tests/e2e/util.go

```diff
@@ -7190,81 +7219,101 @@ func validate_totalStoragequota(ctx context.Context, diskSize int64, totalUsedQu
 
 	ginkgo.By(fmt.Sprintf(" quotaAfter :  %v%s", quotaAfter, string(suffix1)))
 
+	totalDiskStorage = sumupAlltheResourceDiskUsage(diskSizes, suffix1)
+
 	if string(suffix) != string(suffix1) {
 		if string(suffix1) == "Mi" {
-			var bytes int64 = quotaBefore
-			// Convert to Gi
-			//kibibytes := float64(bytes) / 1024
-			quotaBefore = int64(bytes) * 1024
-			fmt.Printf("Converted quotaBefore to Mi :  %dMi\n", quotaBefore)
+			if string(suffix) == "Gi" {
```

Comment:
Please mention in what cases it will come in GB or MB and why we need this calculation.

Reply:
changed  the variable name also added a comment